lib/srv/db/dynamodb/engine.go

```diff
@@ -43,19 +43,19 @@ import (
 	"github.com/gravitational/teleport/lib/cloud/awsconfig"
 	"github.com/gravitational/teleport/lib/defaults"
 	"github.com/gravitational/teleport/lib/events"
-	"github.com/gravitational/teleport/lib/modules"
 	"github.com/gravitational/teleport/lib/srv/db/common"
 	"github.com/gravitational/teleport/lib/srv/db/common/role"
 	"github.com/gravitational/teleport/lib/utils"
 	libaws "github.com/gravitational/teleport/lib/utils/aws"
+	"github.com/gravitational/teleport/lib/utils/aws/dynamodbutils"
 )
 
 // NewEngine create new DynamoDB engine.
 func NewEngine(ec common.EngineConfig) common.Engine {
 	return &Engine{
 		EngineConfig:  ec,
 		RoundTrippers: make(map[string]http.RoundTripper),
-		UseFIPS:       modules.GetModules().IsBoringBinary(),
+		UseFIPS:       dynamodbutils.IsFIPSEnabled(),
```

Comment:
Technically this line doesn't seem required for dynamo to work, at least at a surface level, but it seemed reasonable to include it.

Reply:
I don't quite follow, are you talking about the line pre-change? If yes then I suppose the answer is yes.